src/Util/UrlUtil.spec.js

```diff
@@ -131,6 +131,24 @@ describe('UrlUtil', () => {
         });
       });
     });
+
+    describe('#objectToRequestString', () => {
+      it('is defined', () => {
+        expect(UrlUtil.objectToRequestString).to.not.be(undefined);
+      });
+      it('returns a requestString from an object', () => {
+        const requestString = 'LAYER=OSM-WMS&VERSION=1.3.0&SERVICE=WMS&REQUEST=getLegendGraphic&FORMAT=image/png';
+        const params = {
+          LAYER: 'OSM-WMS',
+          VERSION: '1.3.0',
+          SERVICE: 'WMS',
+          REQUEST: 'getLegendGraphic',
+          FORMAT: 'image/png'
```

Comment:
This test should then use sth. that need urlencoding like `'&'` or `'üöäß'` or `' '`, ...

Reply:
Solved.